optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/util/PlannerTestUtils.java

```diff
@@ -85,6 +89,21 @@ public Score calculateScore(Solution solution) {
 
     }
 
+    public static String readResourceToString(Class clazz, String resource) throws IOException {
```

Comment:
Couldn't find the right utils class. Thanks. Fixed to `IOUtils.toString(getClass().getResourceAsStream(solverConfigResource), "UTF-8");`